runtime/runtime/src/ext.rs

```diff
@@ -359,16 +358,26 @@ impl<'a> External for RuntimeExt<'a> {
     fn get_receipt_receiver(&self, receipt_index: ReceiptIndex) -> &AccountId {
         self.receipt_manager.get_receipt_receiver(receipt_index)
     }
+}
+
+pub(crate) struct RuntimeContractExt<'a> {
```

Comment:
nit. assuming `Ext` suffix represents "external", what does external contract mean here? (I miss the context for the structures in `ext.rs`.

Reply:
That's a good question. "External" here stands for the "External Interface to the Contract Runtime". For this structure it could really just also read "RuntimeContractGetter" or "ContractRuntimeReader" or "Accessor" or something similar to evoke the purpose of this data structure more clearly, but since we already have `RuntimeExt` as a precedent, I opted to go with a similar name here as well.